services/catalog/src/simcore_service_catalog/core/application.py

```diff
@@ -64,6 +65,10 @@ def create_app(settings: ApplicationSettings | None = None) -> FastAPI:
         app.add_middleware(
             BaseHTTPMiddleware, dispatch=timing_middleware.add_process_time_header
         )
+    if app.state.settings.CATALOG_TRACING:
+        setup_opentelemetry_instrumentation(
+            app, app.state.settings.CATALOG_TRACING, "simcore_service_catalog"
+        )
 
```

Comment:
so I think everywhere where you do
setup_opentelemetry_instrumentation(
            app, app.state.settings.CATALOG_TRACING, "simcore_service_catalog"
        )

you can probably replace the APP_NAME with APP_NAME.